### DISCUSSION TITLE
Code review ІМ-13

### DISCUSSION THREAD

**Discussion by Dimdim28** [HowProgrammingWorks/LiveQA#44] [Code review]
Пишу браузерну версію гри: "танки".
У Projects описав, що вже зроблено, а що ні.
Суть гри : обрати техніку та знищувати ворогів, що з'являться. За кожного знищенного ворога гравець отримує певну кількість очок за які потім може обрати кращу техніку.  Після смерті очки не зникаюсь, але якщо їх стане недостатньо для вибору нової техніки, гра закінчується. Скоріш за все буде реалізована нескінченна кількість раундів у грі.
Створив репозиторій у який потім на інші гілки  додадуть  свої ігри люди, з якими ще будемо рефакторити мій та їх код.
Опис поки не робив. Буду вдячній за ревью [кода](https://github.com/Dimdim28/Webgames).

**Comment by tshemsedinov** [comment]
See code review here: https://github.com/Dimdim28/Webgames/issues/1

**Reply by tshemsedinov** [reply]
- Use linter and formatter before commit
- Use intermediate variables: https://github.com/Dimdim28/Webgames/blob/ab1e8865a7c798a2ac03cd6c7ee5215f3ca2638a/tanks.js#L118

**Comment by tshemsedinov** [comment]
- Don't use ternary instead of if-statement https://github.com/Dimdim28/Tanks.js/blob/b65f7d880f78a298640ee3b6f29fe10a187b2828/tanks.js#L175-L180
- Strange syntax `(player.x = GAMEZONEWIDTH / 2), (player.y = GAMEZONEHEIGHT / 2), init();`
- Combine fns to a single one https://github.com/Dimdim28/Tanks.js/blob/b65f7d880f78a298640ee3b6f29fe10a187b2828/tanks.js#L235-L241
